pkgs/jnigen/test/simple_package_test/runtime_test_registrant.dart

```diff
@@ -742,7 +742,7 @@ void registerTests(String groupName, TestRunnerCallback test) {
           // listener, it will work.
           runner.runOnAnotherThreadAndJoin();
           await completer.future;
-          expect(MyRunnable.$impls, hasLength(1));
+          expect(MyRunnable.$impls, hasLength(1), skip: Platform.isAndroid);
```

Comment:
Maybe add some documentation why this needs to be skipped?

Reply:
The docs are below it, will move it up.